test/gurka/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 if(ENABLE_DATA_TOOLS)
   file(GLOB_RECURSE TEST_FILES "${CMAKE_CURRENT_LIST_DIR}/test_*.cc")
-
+  add_definitions(-Dgurka_tests)
```

Comment:
Why this define?

Reply:
I will revert that change.  I was originally playing with some preprocessing directives that would be used for testing the constraints, but that is no longer needed.